ngrinder-frontend/src/js/main.js

```diff
@@ -110,7 +110,14 @@ Vue.directive('visible', (el, binding) => {
 Vue.filter('numFormat', numFormat(numeral));
 Vue.filter('dateFormat', (value, format) => {
     if (value) {
-        return moment(new Date(value)).tz(ngrinder.currentUser.timeZone).format(format);
+        let date = new Date(0);
```

Comment:
Isn't ```new Date(value)``` enought?

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Date

Reply:
@junoyoon 

Yes, It works fine... sorry
It didn't work when I tried it before. Maybe I didn't refresh the UI properly T.T

Fixed.
https://github.com/naver/ngrinder/blob/b0a2e7756b676d3614d397c5cddffdc470efd518/ngrinder-frontend/src/js/main.js#L113